lib/server/routes/coinpayments.js

```diff
@@ -132,10 +167,10 @@ CoinpaymentsRouter.prototype._definitions = function () {
     ['POST', '/coinpayments',
       bodyParser.json(),
       bodyParser.urlencoded({ extended: true }),
-      Coinpayments.ipn({
-        'merchantId': process.env.CP_MERCHANT_ID,
-        'merchantSecret': process.env.CP_IPN_SECRET
-      }),
+      //Coinpayments.ipn({
```

Comment:
do we need to keep this?

Reply:
It's commented out for testing, uncommenting it for staging push.